libs/order/src/drivers/src/magento/errors/codes.ts

```diff
@@ -0,0 +1,3 @@
+export enum MagentoOrderGraphQlErrorCode {
+	CART_NOT_FOUND = 'graphql-no-such-entity',
```

Comment:
Could we reuse the error code `@daffodil/cart`? It's the same code

Reply:
I don't think it gets exported.